third_party/2/google/protobuf/any_test_pb2.pyi

```diff
@@ -0,0 +1,39 @@
+# @generated by generate_proto_mypy_stubs.py.  Do not edit!
+from typing import (
+    Iterable,
+)
+
+from google.protobuf.any_pb2 import (
+    Any,
+)
+
+from google.protobuf.internal.containers import (
+    RepeatedScalarFieldContainer,
+)
+
+from google.protobuf.message import (
+    Message,
+)
+
+
+class TestAny(Message):
```

Comment:
probably not much value in annotating the test protos

Reply:
Probably not much value, sure.  But is there any reason to **not** include stubs for all of the generated python files, since it's easy?

Reply:
nah, no strong reason. just a few more bytes for everyone to pull down :) not a big deal